src/components/themed/ExchangedFlipInput2.tsx

```diff
@@ -204,7 +204,7 @@ const ExchangedFlipInput2Component = React.forwardRef<ExchangedFlipInputRef, Pro
 
   return coreWallet != null ? (
     <>
-      <TouchableOpacity onPress={headerCallback} style={styles.headerContainer}>
+      <TouchableOpacity accessible={false} onPress={headerCallback} style={styles.headerContainer}>
```

Comment:
What children were made accessible here?

Reply:
The EdgeText is made accessible